code/Test_definitions/device-roaming-status-subscriptions.feature

```diff
@@ -160,66 +160,65 @@ Feature: Device Roaming Status Subscriptions API, v0.6.0 - Operations RoamingSta
     Then the response code is 400
     And the response property "$.status" is 400
     And the response property "$.code" is "INVALID_ARGUMENT"
     And the response property "$.message" contains a user friendly text
 
- @roaming_status_subscription_16_invalid_protocol
-   Scenario: subscription creation with invalid protocol
+  @roaming_status_subscription_16_invalid_protocol
+  Scenario: subscription creation with invalid protocol
     Given a valid subscription request body 
     And the request property "$.protocol" is not "HTTP"
     When the request "createDeviceRoamingStatusSubscription" is sent
     Then the response property "$.status" is 400
     And the response property "$.code" is "INVALID_PROTOCOL"
     And the response property "$.message" contains a user friendly text
 
-@roaming_status_subscription_17_invalid_credential_type
-   Scenario: subscription creation with invalid credential type
+  @roaming_status_subscription_17_invalid_credential_type
+  Scenario: subscription creation with invalid credential type
     Given a valid subscription request body 
     And the request property "$.credentialType" is not "ACCESSTOKEN"
     When the request "createDeviceRoamingStatusSubscription" is sent
     Then the response property "$.status" is 400
     And the response property "$.code" is "INVALID_CREDENTIAL"
     And the response property "$.message" contains a user friendly text
 	
-@roaming_status_subscription_18_invalid_access_token_type
-   Scenario: subscription creation with invalid access token type 
+  @roaming_status_subscription_18_invalid_access_token_type
+  Scenario: subscription creation with invalid access token type
     Given a valid subscription request body 
     And the request property "$.accessTokenType" is not "bearer"
     When the request "createDeviceRoamingStatusSubscription" is sent
     Then the response property "$.status" is 400
     And the response property "$.code" is "INVALID_TOKEN" or "INVALID_ARGUMENT"
     And the response property "$.message" contains a user friendly text
 
-@roaming_status_subscription_19_invalid_credentials
-   Scenario: subscription creation with invalid credentials
+  @roaming_status_subscription_19_invalid_credentials
+  Scenario: subscription creation with invalid credentials
     Given a valid subscription request body 
     And header "Authorization" token is set to invalid credentials
     When the request "createDeviceRoamingStatusSubscription" is sent
     Then the response property "$.status" is 401
     And the response property "$.code" is "UNAUTHENTICATED"
     And the response property "$.message" contains a user friendly text
 
-@roaming_status_subscription_20_invalid_inconsistent_access_token
-   Scenario: subscription creation with inconsistent access token for requested events subscription
+  @roaming_status_subscription_20_invalid_inconsistent_access_token
+  Scenario: subscription creation with inconsistent access token for requested events subscription
```

Comment:
Same comment than previously regarding SUBSCRIPTION_MISMATCH

Reply:
Thanks for the review - revision is on the way.